apps/prairielearn/src/ee/lib/ai-grading/ai-grading.ts

```diff
@@ -160,6 +161,10 @@ export async function aiGrade({
       instance_question: InstanceQuestion,
       logger: AIGradingLogger,
     ) => {
+      const grading_jobs = gradingJobMapping[instance_question.id] ?? [];
+      const manualGradingJob = grading_jobs.find((job) => job.grading_method === 'Manual');
+      const should_update_score = !manualGradingJob;
```

Comment:
Similar comment as other PR: typically we save `snake_case` for things that are database tables/columns; this is neither.

Similar comment as above: `Array.prototype.some` is better here.


```suggestion
      const shouldUpdateScore = gradingJobMapping[instance_question.id]?.some((job) => job.grading_method === 'Manual');
```

(+/- some Prettier reformatting)

Final note: please add a comment explaining/emphasizing what's going on here. Here's an option:

> We always want human grading to take precedence, so we'll only update the score for instance questions that haven't previously been graded by a human.

Reply:
Done in 758a8a0daab2976a08fc968d6c12758453d648b4.